engine/security/src/main/scala/pl/touk/nussknacker/ui/security/basicauth/BasicHttpAuthenticator.scala

```diff
@@ -7,10 +7,9 @@ import pl.touk.nussknacker.engine.util.cache.DefaultCache
 import pl.touk.nussknacker.ui.security.api.{DefaultAuthenticationConfiguration, LoggedUser, RulesSet}
 import pl.touk.nussknacker.ui.security.basicauth.BasicHttpAuthenticator.{EncryptedPassword, PlainPassword, UserWithPassword}
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.Future
+import scala.concurrent.{ExecutionContext, Future}
 
-class BasicHttpAuthenticator(configuration: DefaultAuthenticationConfiguration, allCategories: List[String]) extends SecurityDirectives.AsyncAuthenticator[LoggedUser] {
+class BasicHttpAuthenticator(configuration: DefaultAuthenticationConfiguration, allCategories: List[String])(implicit ec: ExecutionContext) extends SecurityDirectives.AsyncAuthenticator[LoggedUser] {
```

Comment:
I think here we don't need ec at all, just replace Future { ... } with Future.successful

Reply:
Removed